app/javascript/scavenger_hunt/data/staticData/clues.js

```diff
@@ -1,21 +1,92 @@
 export const Clues = [
```

Comment:
If we hand-code all the questions, then we don't need this file right? This is just a referrence.

Reply:
I will keep it for now. Will remove it later once all Questions are done

Reply:
cool